hubploy/utils.py

```diff
@@ -1,5 +1,6 @@
 """
-Utilities for calling out to git
+Utils to extract information (last_modified_commit, path_touched) from the git
+history.
```

Comment:
I don't mention the other functions as they act as helper functions to the two mentioned.